.github/workflows/js-lints.yml

```
@@ -0,0 +1,59 @@
+name: JavaScript Lints
```

Minor change as this seems to also lint SCSS, perhaps this should read:

```suggestion
name: JavaScript and SASS Lints
```

---

Why not "SCSS"?